packages/deployer/src/utils/contract.ts

```diff
@@ -28,16 +28,16 @@ export class Contract implements Web3.ContractInstance {
         _.forEach(functionsAbi, (functionAbi: Web3.MethodAbi) => {
             if (functionAbi.constant) {
                 const cbStyleCallFunction = this._contract[functionAbi.name].call;
-                this[functionAbi.name] = {
-                    callAsync: promisify(cbStyleCallFunction, this._contract),
```

Comment:
Why did we get rid of `callAsync` and re-instate the use of `call`? It's still an async call no?

Reply:
It is. Will fix that in abi-gen PR